packages/docs-sidenav/index.js

```diff
@@ -189,7 +189,7 @@ function NavTree({ baseRoute, content }) {
       )
     }
     // Individual pages (leaf nodes)
-    if (item.path) {
+    if (typeof item.path == 'string') {
```

Comment:
We now support an empty string path `""` for the index page, so had to modify these checks slightly.